packages/terra-worklist-data-grid/src/proptypes/columnHeaderShape.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import columnHeaderCellShape from './columnHeaderCellShape';
+
+const columnHeaderShape = PropTypes.shape({
+  /**
+   * String that specifies the column header height. Any valid CSS height value is accepted.
+   */
+  height: PropTypes.string,
```

Comment:
This property should be part of the main data grid.  It would not be part of the column header shape.

Reply:
Agreed and removed.